src/aleph_client/commands/aggregate.py

```diff
@@ -0,0 +1,40 @@
+import typer
+from typing import Optional
+from aleph_client.types import AccountFromPrivateKey
+from aleph_client.account import _load_account
+from aleph_client.conf import settings
+from pathlib import Path
+from aleph_client import synchronous
+from aleph_client.commands import help_strings
+
+from aleph_client.commands.message import forget_messages
+
+from aleph_client.commands.utils import setup_logging
+
+from aleph_message.models import MessageType
+
+app = typer.Typer()
+
+@app.command()
+def forget(
```

Comment:
same name as in the aggregate.py file

Reply:
Same here, 

`Since it is a subcommand, it would look like 'aleph program upload'. Therefore there is no overlap between command names, as this function is in a different file assiociated with a different command ;)`

So no need to change names, all good.